cypress/integration/1221-my_publication/576-creator_cancels_deletion_of_an_item_in_my_registrations_list.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Creator cancels deletion of an item in My Registrations list
+
+    @576
+    Scenario: Creator cancels deletion of an item in My Registrations list
+        Given that the user is logged in as Creator
+        And is on the My Registrations page

```

Comment:
They is? 😅 gått litt fort i svingene her kanskje. Ikke din feil da, men scenarioet burde blitt oppdatert.